app/code/Magento/AdvancedSearch/Test/Mftf/ActionGroup/AdminSaveSearchTermActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSaveSearchTermActionGroup">
+        <annotations>
+            <description>Save a new search term from Magento admin.</description>
+        </annotations>
+        <!-- Click save action and verify success message -->
+        <click selector="{{AdminMainActionsSection.save}}" stepKey="clickSaveSearchButton"/>
+        <see userInput="You saved the search term." stepKey="assertSaveSearchTermSuccessMessage" />
```

Comment:
Please, do not combine assertion and saving in the same action group. Instead, leave only saving process in the current action group and use the [existing](https://github.com/magento/magento2/blob/83e3487a12e1165dde0c9175fbad45b9bfe018c3/app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertMessageInAdminPanelActionGroup.xml#L11) action group for the assertion

Reply:
@rogyar Thanks for the review. I have used the existing action group for Assertion. Please review it.